scripts/deps-ppa/static_z3.sh

```diff
@@ -73,14 +73,14 @@ cp "/tmp/${packagename}_${debversion}.orig.tar.gz" ../
 # Create debian package information
 
 mkdir debian
-echo 9 > debian/compat
+echo 13 > debian/compat
 # TODO: the Z3 packages have different build dependencies
 cat <<EOF > debian/control
 Source: z3-static
 Section: science
 Priority: extra
 Maintainer: Daniel Kirchner <daniel@ekpyron.org>
-Build-Depends: debhelper (>= 9.0.0),
+Build-Depends: debhelper (>= 13.0.0),
                cmake,
                g++ (>= 5.0),
```

Comment:
Why not use `>= 9.0` here too?

btw @ekpyron the [documentation](https://docs.soliditylang.org/en/v0.8.17/installing-solidity.html) still recommends gcc 8 as the minimum